tasks/models.py

```diff
@@ -25,6 +26,15 @@ class Task(models.Model):
     status = EnumChoiceField(Status, default=Status.BACKLOG, max_length=1)
     description = models.TextField(null=True, blank=True)
 
+    @classmethod
+    def update_status(task_id, status):
+        task_id.status = status
+
+    @classmethod
+    def update_priority(task_id, priority):
```

Comment:
Since it is a class method, add `cls` as the first argument.
`def update_priority(cls, task_id, priority):`
Otherwise the function won't work properly

Reply:
I would go with `@staticmethod`